components/Order/adjustments.rst

```diff
@@ -48,28 +19,20 @@ Every **Adjustment** instance has the ``neutral`` property, which indicates if i
 
     $order = new Order();
     $tshirt = new OrderItem();
-    $tshirt
-        ->setName('Awesome T-Shirt')
-        ->setUnitPrice(4999)
-    ;
+    $tshirt->setUnitPrice(4999);
 
     $shippingFees = new Adjustment();
     $shippingFees->setAmount(1000);
 
     $tax = new Adjustment();
-    $tax
-        ->setAmount(1150)
-        ->setLabel()
-        ->setNeutral(true)
-    ;
-
-    echo $order
-        ->addItem($tshirt)
-        ->addAdjustment($shippingFees)
-        ->addAdjustment($tax)
-        ->calculateTotal()
-        ->getTotal()
-    ;
+    $tax->setAmount(1150);
+    $tax->setNeutral(true);
+
+    $order->addItem($tshirt);
+    $order->addAdjustment($shippingFees);
+    $order->addAdjustment($tax);
```

Comment:
Insert a blank line to increase readability. This applies also to other adjustment examples.